### PR TITLE
The Quartermaster's Requisition Digi-board

### PR DESCRIPTION
**Pull request by Endecc**
## About the PR

Gave the quartermaster a new item in their locker, a chunky clipboard filled with details of Nanotrasen Finance details, Shipping Manifests, and the heavy, concussive weight of Bureaucracy. But enemy corporations like the idea of acquiring Nanotrasen's banking details in a single step, so QMs will have a bit more pressure applied to their department!

## Why / Balance

While Quartermaster had a steal objective associated with them in the past with the Cargo Shuttle board, it was trivial to get by stealing it from the cargo shuttle itself. This was removed because it was a massive annoyance to the whole station when it was taken that way, because it left the cargo shuttle often completely inoperable in an empty part of space, as traitors took it to the void, leaving resupplies for the station crippled.

So what other special item could the Quartermaster be given, relevant to their role, but not extremely impactful to the events of the whole station? A clipboard! A bureaucracy enthusiast's dream tool, with sturdy pins for holding many more of the shipping manifests that are accumulated from cargo orders.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
https://www.youtube.com/watch?v=kyofGodS_8w

**Changelog**

:cl:
- add: Added the Requisition Digi-board, a wireless cargo computer (and high-capacity clipboard) for the Quartermaster's Locker.
- add: Added a new Steal objective for traitors, as this new fancy clipboard has access to most of Nanotrasen's accounting details!

### CODE REVIEW

**Comment by Eden077** [comment]
The sprite could use some downsizing imo, otherwise it looks good

**Comment by deltanedas** [comment]
for the record its the request board that used to be an objective but it was bad

**Comment by Eden077** [comment]
Remember! Mix up all the orders so if they do catch you and return it to QM, the orders are never the same

**Comment by Endecc** [comment]
> The sprite could use some downsizing imo, otherwise it looks good

It was hard to get it to look like as much of a chunky brick without making it, well, chunky. I don't think I could downsize it without losing the digital aspect it has, but if it's called necessary then I can try.

> for the record its the request board that used to be an objective but it was bad

whoops

> Remember! Mix up all the orders so if they do catch you and return it to QM, the orders are never the same

This is high level antagonist gameplay, if this was the fate of my clipboard I would be devastated

File: Resources/Textures/Objects/Misc/qm_clipboard.rsi/meta.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Clipboard sprites are by Endecc (Github) for SS14 Wizard's Den, taken from ***NO LINK YET, ME***, qm_clipboard_paper is a modified version of paper from bureaucracy.rsi.",
+  "copyright": "Clipboard sprites are by Endecc (Github) for SS14 Wizard's Den, taken from https://github.com/space-wizards/space-station-14/pull/21023, qm_clipboard_paper is a modified version of paper from bureaucracy.rsi.",
```
Comment (deltanedas) [inline Resources/Textures/Objects/Misc/qm_clipboard.rsi/meta.json:4]: just say its for SS14 dont need to link to your own pr :trollface:

Comment (Endecc) [reply Resources/Textures/Objects/Misc/qm_clipboard.rsi/meta.json:4]: I probably misunderstood the practice of just linking external PRs, but it probably doesn't hurt to have there.
But I am clueless, I can take it out if necessary to merge

Comment (deltanedas) [reply Resources/Textures/Objects/Misc/qm_clipboard.rsi/meta.json:4]: im not maint :trollface: but ive never seen anyone else do it

Comment (Endecc) [reply Resources/Textures/Objects/Misc/qm_clipboard.rsi/meta.json:4]: I think I've done it for every other one of my sprite-adding PRs without issue, unless I'm gaslighting myself
We will see what a maint thinks of it if it's a problem

**Comment by LankLTE** [comment]
Not sure how I feel about this as a traitor objective as the cargo request board was like, widely considered the worst objective in the game and I feel this is similar. The only possible difference is that the QM might be carrying it on them- but a majority of the other “kill head” objectives also have some other use (like the hypo, nuke disk, or hand teleporter) while this doesn’t have a huge incentive for traitors to take it or for the QM to carry it as it doesn’t have an *actual* use (I know it can store papers- I’m speaking from a standpoint that it is essentially a glorified folder as far as practical use goes).

**Comment by GoodWheatley** [comment]
> but a majority of the other “kill head” objectives also have some other use (like the hypo, nuke disk, or hand teleporter)

If the emergency security orders can be a steal objective, this can be a steal objective

**Comment by CatMagic1** [comment]
what use does the nuke disk have?

**Comment by metalgearsloth** [comment]
> what use does the nuke disk have?

the nuke?

**Comment by CatMagic1** [comment]
> > what use does the nuke disk have?
> 
> the nuke?

syndies don't have nuke codes

but whatever, unrelated

**Comment by dvir001** [comment]
Size 30?
Capacity 100?

Seem like an over kill, should be more like size 10 capacity 30 😬

**Comment by Endecc** [comment]
> Damage for it should probably be lowered slightly, but besides that, looks good imo.

Damage can be tweaked, sure. I noticed the clipboard dealt 6 damage, so I just doubled that. Will lower to 10.

> Size 30? Capacity 100?
> 
> Seem like an over kill, should be more like size 10 capacity 30 😬

I'm up for tweaking some numbers, but size 10 capacity 30 is barely better storage than a _normal_ clipboard (at 20 storage), and is the _exact same weight_. My intent was to make it feel as bulky as it goes out to say it is, but making it worth the upgrade for people who really like their paperwork.
It's also an objective item that is comparatively speaking, pretty easy to steal. It should have some drawback of being a bit more difficult to store on your person discreetly.

> while this doesn’t have a huge incentive for traitors to take it or for the QM to carry it as it doesn’t have an _actual_ use (I know it can store papers- I’m speaking from a standpoint that it is essentially a glorified folder as far as practical use goes).

What I had left out when implementing were any of the extra ideas I had for it.  AoE paper grabbing like a garbage bag, or increasing in damage (probably up to a cap) the more papers it had. An idea I had just now actually is a wireless cargo request computer, which would be useful if the department is unpowered. Do any of these extra ideas yourself, Lank?

**Comment by Endecc** [comment]
Make the digi-board digitally compatible is the plan, then. Time to figure out how to make an item a computer.

**Comment by Endecc** [comment]
Don't think the failed check is my fault, calling this ready to review

**Review by LankLTE — APPROVED** [review]
The objective may need to be tweaked in the future but overall I'm fine with it

**Pull request merged**